src/index.ts

```diff
@@ -1,1220 +1,7 @@
-// @ts-ignore
-import {PurchasesError, PurchasesErrorCode, PurchasesErrorHelper} from './errors';
-import { NativeEventEmitter, NativeModules, Platform } from "react-native";
+import Purchases from './purchases';
+export default Purchases;
 
 export * from './errors';
```

Comment:
probably easier to understand side-by-side. Really it just moves everything out of this file and then the file exports everything, so that `index.ts` acts as... well, an index.